site/assets/css/inline.css

```diff
@@ -72,6 +72,7 @@ body {
   border-bottom: 1px solid rgba(0, 0, 0, 0.1);
   display: flex;
   justify-content: space-between;
+  z-index: 99999;
```

Comment:
z-index added so the navbar stays above the youtube videos now that we have content ~ab~using `position: absolute` :-)